### ISSUE TITLE
Ability to "attach" sources not discoverable by Maven

### ISSUE THREAD

**Issue by shevek** [kelemen/netbeans-gradle-project#112]
I am using some JARs for which the sources are available, but not in Maven.

Is it possible to "attach" the sources somehow by selecting an external JAR file?

This is already possible for Netbeans platforms and JDKs.

**Comment by jhanduber** [comment]
I'm not sure how else to upvote this issue, soo...upvote! It would be great to have this feature. Constantly switching over the javadocs for 3rd party libraries in the browser is a bit...pre-maven'ish!

**Comment by JustGregory** [comment]
@jhanduber I agree with the sentiment, based upon the importance of having sources available; but in lieu of "upvoting", I'd personally much rather see the general volume of comments on issues increase, as well as the accuracy on details improve. I'm not meaning necessarily for this repo, but across the whole Github site.
